universal_silabs_flasher/const.py

```diff
@@ -68,4 +69,12 @@ class ResetTarget(enum.Enum):
         },
         "toggle_delay": 0.1,
     },
+    ResetTarget.SLZB07: {
+        "chip_name": "cp210x",
```

Comment:
Is this config stable if a CP210x chip is plugged in over USB?

Reply:
This is exactly the purpose to detect the CP210x plugged in via USB as we dont otherwise know the `gpiochipX` number in advance.  If more than one cp210x device (cp2012N and above only, plain cp2102 doesnt have GPIO) is plugged in, it might return the wrong one, but the api doesnt provide anyway to detect via USB info. I guess in this case just need to have documentation warning user to ensure only single device is plugged in, when using the alzb-07 reset command.

If the wrong chip is picked up, it is going to switch the GPIO on that chip, which will probably do nothing as no other dongles I know of implement these gpio's (yet).

Reply:
I have added a message when using USB gpio reset to ensure no other devices are connected. Given this option would only be used in extraordinary circumstances, broken or router firmware installed, this should be sufficient?